src/Umbraco.Tests.Integration/TestServerTest/UmbracoWebApplicationFactory.cs

```diff
@@ -20,7 +20,7 @@
 using Umbraco.Core.Models.Membership;
 using Umbraco.Core.Services;
 using Umbraco.Web.Common.Security;
-using Umbraco.Web.UI.BackOffice;
+using Umbraco.Web.UI.NetCore;
 
 namespace Umbraco.Tests.Integration.TestServerTest
```

Comment:
This change slipped into this PR 😁

Reply:
This is expected.. The namespace of startup.cs and program.cs was wrong.